addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/welcome/NAWelcomePersonHandler.java

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.handler.welcome;
+
+import static org.openhab.binding.netatmo.NetatmoBindingConstants.*;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.netatmo.config.NetatmoWelcomeConfiguration;
+
+import io.swagger.client.model.NAWelcomeEvents;
+import io.swagger.client.model.NAWelcomePersons;
+
+/**
+ * {@link NAWelcomePersonHandler} is the class used to handle the Welcome Home Data
+ *
+ * @author Ing. Peter Weiss - Welcome camera implementation
+ *
+ */
+public class NAWelcomePersonHandler extends AbstractNetatmoWelcomeHandler {
+
+    private NetatmoWelcomeConfiguration configuration;
+    protected NAWelcomePersons person;
+    private NAWelcomeEvents event;
+
+    public NAWelcomePersonHandler(Thing thing) {
+        super(thing);
+        this.configuration = this.getConfigAs(NetatmoWelcomeConfiguration.class);
```

Comment:
It might be required to set "configuration" in initialize() rather than the constructor.